README.md

```diff
@@ -239,6 +239,31 @@ export default class World extends ActorWorld {
 setWorldConstructor(World)
 ```
 
+If you are using ES Modules (by setting `"type": "module"` in `package.json`), you need to
+change ypur `World` constructor, and add a `Begin` step:
```

Comment:
What's a `Begin` step? Do you mean a `Before` hook?

Reply:
yes, I did mean Before.